package.json

```diff
@@ -96,14 +92,14 @@
     "@types/jest-environment-puppeteer": "^5.0.0",
     "@types/lodash": "^4.14.176",
     "@types/markdown-it": "^12.2.3",
-    "@types/puppeteer": "^5.4.5",
+    "@types/puppeteer": "^5.4.4",
     "@types/react": "^17.0.15",
     "@types/react-dom": "^17.0.11",
     "@types/react-router-dom": "^5.3.3",
     "@types/shelljs": "^0.8.9",
     "@types/uuid": "^8.3.1",
     "@typescript-eslint/eslint-plugin": "^5.12.1",
-    "@typescript-eslint/parser": "^5.12.1",
+    "@typescript-eslint/parser": "^5.11.0",
```

Comment:
Why do you have multiple packages stepping backwards on versions? I wonder if that's the prob w frozen-lockfile

Reply:
I stepped this back to see was this the issue, as I merged the latest develop into this branch (which had the updated version, so I thought this may be the issue but it wasn't). I'll update this to the latest develop versions, but I don't think it resolves the problem.